NOTICE

```diff
@@ -1,4 +1,8 @@
-Datadog .NET Tracer for APM (dd-trace-dotnet)
-Copyright 2017 Datadog, Inc.
+Application Telemetry & Performance Monitoring Product Suite for .NET Applications
+Datadog 
+
+(dd-trace-dotnet)
+
+Copyright 2017-2021 Datadog, Inc.
```

Comment:
Can you revert this back to `Copyright 2017 Datadog, Inc`? We don't need to include the end year for it to take effect, so I'd prefer removing it

Reply:
Addressed in https://github.com/DataDog/dd-trace-dotnet/commit/8b9642541f0ad21873f3bdd9325b6601a10bb8c9